src/themes/admin_future/modules/voting/setting.tpl

```diff
@@ -0,0 +1,24 @@
+<!-- BEGIN: main -->
+<div class="card border-primary border-3 border-bottom-0 border-start-0 border-end-0">
+    <div class="card-body pt-4">
+        <form method="post" class="ajax-submit" action="{$smarty.const.NV_BASE_ADMINURL}index.php?{$smarty.const.NV_LANG_VARIABLE}={$smarty.const.NV_LANG_DATA}&amp;{$smarty.const.NV_NAME_VARIABLE}={$MODULE_NAME}&amp;{$smarty.const.NV_OP_VARIABLE}={$OP}" novalidate>
+            <div class="row mb-3">
+                <label class="col-sm-5 col-lg-3 col-form-label text-sm-end">
```

Comment:
@nambmtk1993 thẻ label nếu không có for Google sẽ báo
```
No label associated with a form field
```
Cho nên mọi label em cần có for="ID_INPUT" trong trường hợp không có ID_INPUT nào phù hợp để gán thì em thay bằng thẻ DIV